site/en/blog/lighthouse-11-0/index.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: 'layouts/blog-post.njk'
+title: What's new in Lighthouse 11
+authors:
+  - adamraine
+  - jasmineyan
+date: 2023-08-24
+description: >
+  Lighthouse 11 is here with new accessibility changes and audit updates
+hero: 'image/MtjnObpuceYe3ijODN3a79WrxLU2/SM05YIr19Ucx2cUCx5lR.png'
+alt: 'Lighthouse category scores, all 100'
+tags:
+  - new-in-lighthouse
+  - lighthouse
+  - chrome-118
+---
+
+[Lighthouse is a website auditing tool](/docs/lighthouse/overview/) that helps developers with opportunities and diagnostics to improve the user experience of their sites.
+
+Lighthouse 11 is available immediately on the [command line through npm](https://www.npmjs.com/package/lighthouse) and in [Chrome Canary](https://www.google.com/chrome/canary/). It will land in Chrome stable in Chrome 118 and in [PageSpeed Insights](https://pagespeed.web.dev/) in the coming weeks.
+
+See the full list of changes in the [11.0 changelog](https://github.com/GoogleChrome/lighthouse/releases/tag/v11.0.0).
+
+## Accessibility Category Updates
```

Comment:
```suggestion
## Accessibility category updates
```
down with title case! :)

(actually up to you, but should make the section headings consistent)